appdata.patch

```diff
@@ -0,0 +1,11 @@
+--- a/org.kde.kstars.appdata.xml	2020-07-18 14:39:01.000000000 +0700
++++ b/org.kde.kstars.appdata.xml	2020-09-15 11:54:12.639581651 +0700
+@@ -470,4 +470,8 @@
+     <value key="KDE::windows_store">https://www.microsoft.com/store/apps/9pprz2qhlxtg</value>
+     <value key="KDE::google_play">https://play.google.com/store/apps/details?id=org.kde.kstars.lite</value>
+   </custom>
++  <releases>
```

Comment:
Is this patch upstream yet?

Reply:
https://invent.kde.org/education/kstars/-/merge_requests/80 only for `content_rating`